flink-streaming-connectors/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/config/ConsumerConfigConstants.java

```diff
@@ -53,6 +56,9 @@ public SentinelSequenceNumber toSentinelSequenceNumber() {
 	/** The initial position to start reading Kinesis streams from (LATEST is used if not set) */
 	public static final String STREAM_INITIAL_POSITION = "flink.stream.initpos";
 
+	/** The initial timestamp to start reading Kinesis stream from (when AT_TIMESTAMP is set for STREAM_INITIAL_POSITION */
+	public static final String STREAM_INITIAL_TIMESTAMP = "flink.stream.init.timestamp";
```

Comment:
What do you think about renaming this to `flink.stream.initpos.timestamp`, instead of `flink.stream.init.timestamp`?
I personally think it's a good idea to do so, because essentially `STREAM_INITIAL_TIMESTAMP` is a sub-setting of `STREAM_INITIAL_POSITION`. In other words, `STREAM_INITIAL_TIMESTAMP` will be meaningless is a `STREAM_INITIAL_POSITION = AT_TIMESTAMP` isn't set in the config.